schemas/update/update_extension_schema_v1.json

```diff
@@ -4,56 +4,89 @@
     "moveStep": {
       "type": "object",
       "properties": {
-        "action": { "const": "move" },
+        "action": {
+          "const": "move",
+          "description": "The action type, which is 'move' for this step"
+        },
         "file": {
+          "description": "The file referenced in this step, either as a string or an object with 'source' and 'target' properties",
           "oneOf": [
-            { "type": "string" },
+            {
+              "type": "string",
+              "description": "The relative path of the file to move the key-value pair"
```

Comment:
```suggestion
              "description": "The relative path of the file in the theme folder to move the key-value pair"
```

Reply:
To clarify this is the relative path to the root of the theme folder not where it might be located on their machine. (same for any other description below that has reference to the "relative path")
If you think it's clear, feel free to ignore this.

Reply:
Good catch! Thank you, @bahar-p!